rfc-139-enable-http3.md

```diff
@@ -0,0 +1,135 @@
+
+### Visual filmstrip
+Below we can see a comparison of the page load under different packet loss conditions using HTTP/2. The top is the baseline, with 0% packet loss, progressing all the way down to 3% packet loss:
+
+![Comparing the visual filmstrip of the effect of packet loss on a page load](rfc-139/visual-filmstrip.png)
+
+As you can see, the visually complete metric sits at 9.5 seconds with no packet loss, but jumps to 13.5 seconds with 3% packet loss. This is a 42% increase.
+
+### Visual progress graph
+The visual progress graph clearly shows the effect packet loss has on page load times. As the packet loss increases, the page load slows down:
+
+![The visual progress graph comparing the percentage completeness of a page over time.](rfc-139/visual-progress.png)
+
+HTTP/3 + QUIC is designed to help improve the page load speed for users experiencing these connection conditions in the real world. 
+
+### Connection quality measurement
+Is it possible to measure the connection quality that our users are experiencing when visiting GOV.UK. Fastly provide a [range of Varnish Configuration Language (VCL) options](https://developer.fastly.com/reference/vcl/variables/client-connection/) that we can use to monitor the quality of user connections should we wish to capture this data:
+
+- [client.socket.ploss](https://developer.fastly.com/reference/vcl/variables/client-connection/client-socket-ploss/) - An estimate of the packet loss on the current connection. Provides a ratio from 0-1. 0 = no loss.
+- [client.socket.tcpi_rtt](https://developer.fastly.com/reference/vcl/variables/client-connection/client-socket-tcpi-rtt/) - The round trip time (RTT) for the current TCP connection to the client. RTT is used for loss detection and congestion control across the connection.
+- [client.socket.tcpi_rttvar](https://developer.fastly.com/reference/vcl/variables/client-connection/client-socket-tcpi-rttvar/) - Measures the variance in RTT of the connection. The higher the variance value, the less stable the connection.
+- [client.socket.tcpi_delivery_rate](https://developer.fastly.com/reference/vcl/variables/client-connection/client-socket-tcpi-delivery-rate/) - Gives a recent value for the delivery bandwidth to the client in bytes per second.
+
+## Our Users
+
+Since HTTP/3 + QUIC offers better performance for users on unstable connections, it is essentially fixing performance for those who were left behind during the transition to HTTP/2. These users are likely to be those who are in the most need of this performance increase. 
+
+Since COVID-19 many people have had to work from home. Many will be in rural areas where high-speed internet access either isn't available, or is expensive. In more populated area many users could still be on ageing cable / DSL connections with [high contention ratios](https://www.broadbandchoices.co.uk/guides/broadband/what-is-contention-ratio). 
```

Comment:
```suggestion
Since COVID-19, many people have had to work from home. Many will be in rural areas where high-speed internet access either isn't available, or is expensive. In more populated areas, many users could still be on ageing cable / DSL connections with [high contention ratios](https://www.broadbandchoices.co.uk/guides/broadband/what-is-contention-ratio). 
```